exasol/slc/internal/tasks/upload/deploy_container_tasks_creator.py

```diff
@@ -7,33 +7,38 @@
     RequiredTaskInfo,
 )
 
+import exasol.slc.internal.tasks.upload.deploy_containers
 from exasol.slc.internal.tasks.upload.deploy_container_task import DeployContainerTask
-from exasol.slc.internal.tasks.upload.upload_containers_parameter import (
-    UploadContainersParameter,
-)
 
 
 class DeployContainerTasksCreator:
 
-    def __init__(self, task: UploadContainersParameter):
+    def __init__(
+        self,
+        task: "exasol.slc.internal.tasks.upload.deploy_containers.DeployFlavorContainers",
```

Comment:
Need to use this annotation because otherwise would have a cyclic dependency
because `DeployFlavorContainers` also imports `DeployContainerTasksCreator`